### ISSUE TITLE
dnsdist failing for rule 'RDRule' with Lua fatal error.

### ISSUE THREAD

**Issue by thechile** [PowerDNS/pdns#5272]
Hi, i'm trying to test dnsdist. Seem to spend most time scratching my head with documentation and hitting cryptic Lua errors like `Fatal error: Exception thrown by a callback function called by Lua` with no pointers to the associated dnsdist.conf related error.

At the moment i'm trying to get RDRule working with `pdndist.conf` configuration like below

```lua
addAction(RDRule(), DropAction())
-- or
addAction(RDRule(), PoolAction("recursive"))
```

But when i try to run i get error

```
/usr/bin/dnsdist --supervised --disable-syslog -v
Read configuration from '/etc/dnsdist/dnsdist.conf'
Added downstream server 10.132.16.15:53
Creating pool recursive
Adding server to pool recursive
Added downstream server 10.132.16.16:53
Adding server to pool recursive
Fatal error: [string "chunk"]:15: attempt to call global 'RDRule' (a nil value)
```
I've read the README several times and no closer to understanding why this error occurs or what it means. 

I'm using `dnsdist-1.1.0-1pdns.el7.x86_64` on `CentOS Linux release 7.3.1611`.

Thank you.

**Comment by thechile** [comment]
ok so `strings /usr/bin/dnsdist | grep RDRule` shows nothing so i assume the latest version from 1.1.x branch does not include this feature. The documentation should allude to what version the different rules/features where introduced or what branch they work in. Having one README, unspecific to a branch/version if very confusing when you assume that what you read will work in the most recent version you just installed.

**Comment by Habbie** [comment]
Yes, agreed. Do you feel like contributing some documentation updates? :)

**Comment by thechile** [comment]
I think the easiest thing for quick win is to update the README to mention at the top of the page that "the documentation tracks master branch and some configuration might not work in earlier stable releases" 

Ideally as each new feature is added the README documentation would include something the below where a footnote expanded on what version its available in and any evolution of the option.  

```
RDRule(): matches queries with the RD flag set[^1][^2]
[^1]: RDRule was added and currently in master
[^2]: RDRule was changed to add option ....
```

Difficulty is tracking when the feature goes from master, tagged into stable release and have documentation automatically reflect this.

**Comment by thechile** [comment]
.. or the same way the pdns authoritative and recursor documentation handles it. 

```
api-logfile

Path
Default: unset
Available since: 4.0.0         <----
```

**Comment by pieterlexis** [comment]
This is fixed on the new [dnsdist.org](https://dnsdist.org/rules-actions.html#RDRule)
